tareas/Erasmo/Tarea2.jl

```diff
@@ -0,0 +1,582 @@
+    c, n = args[1], args[2]
+    n <= 1 && return f(x,c)
+    for it = 1:n
+        x = f(x,c)
+    end
+    return x
+end
+#-
+
+# ## Ejercicio 1:
+# 
+# Para la familia de mapeos cuadrática $Q_c(x) = x^2 + c$, generen el diagrama que
+# muestra el exponente de Lyapunov en términos de $c$, para $c\in[-2,2]$.
+# Utilizen un paso suficientemente fino (en $c$) para que el diagrama muestren
+# la riqueza del comportamiento.
+# 
+# - ¿Qué particularidad ocurre (en términos del exponente de Lyapunov) para los valores de $c$ donde hay bifurcaciones?
+# 
+# - ¿Qué particularidad ocurre (en términos del exponente de Lyapunov) cuando tenemos *ciclos superestables*, es decir, cuando $x=0$ es parte de un ciclo periódico?
+
+#Respuesta
+#-
+
+# ## Ejercicio 2:
+# 
+# Llamaremos $c_n$ al valor del parámetro $c$ donde ocurre el ciclo superestable
+# de periodo $2^n$, esto es, el valor de $c$ donde $x_0=0$ pertenece a la órbita
+# periódica de periodo $2^n$. Algunos de estos valores fueron obtenidos
+# numéricamente en una de las notas de clase.
+# (De manera alternativa, pueden considerar que $c_n$ es el valor del parámetro $c$
+# donde ocurre la bifurcación de doblamiento de periodo para el mapeo $Q_c(x)=x^2+c$,
+# es decir, donde la órbita de periodo $2^n$ nace. Como hemos visto en clase,
+# tenemos que $c_0=1/4$ marca la aparición del atractor de periodo $2^0=1$,
+# $c_1=-1/4$ corresponde a la aparición del atractor de periodo $2^1=2$ y
+# $c_2=-3/4$ a la aparición del atractor de periodo $2^2=4$.)
+# 
+# - Calculen los valores de $c_r$ (al menos hasta $c_6$, pero traten
+# de obtener aún más valores). Con estos valores, definimos la secuencia:
+# $\{f_0, f_1, f_2, \dots\}$, donde
+# \begin{equation}
+# f_n = \frac{c_n-c_{n+1}}{c_{n+1}-c_{n+2}} .
+# \end{equation}
+# Aproximen el valor al que converge esta secuencia,
+# es decir, dar una estimación de $\delta = f_\infty$.
+# 
+# - De los $2^p$ puntos del ciclo de periodo $2^p$, es decir,
+# $\{0, p_1, \dots p_{2^{n-1}}\,\}$ hay uno (distinto del 0) cuya distancia
+# a 0 es la menor; a esa distancia la identificaremos como $d_n$.
+# Estimen numéricamente a qué converge la secuencia $\alpha = - d_n/d_{n+1}$ en
+# el límite de $n$ muy grande.
+
+# ## Solución:
+# 
+# Se sigue la estrategia de clase _41-Universalidad_ para calcular los $c_n$.
+# Esto es, $c_n$ corresponde al valor de $c$ para el cual ocurre el ciclo
+# superestable de periodo $2^n$. De esta forma, $c_n$ es solución a la ecuación
+# de grado $2^n$ dada por
+# 
+# $$ Q_c^{2^n}(0) = 0 .$$
+# 
+# Sea $q^n(c) = Q_c^n(0)$, de tal forma que de tal forma que $c_n$ es una de las
+# raíces de $q^{2^n}(c)$. Nótese que aquí cambió la definición del exponente
+# ya que $q^{n+1}(c) \neq q^1(q^n(c))$.
+# 
+# A continuación se muestra una gráfica de $q^{2n}(c)$ con $n = 0, 1,...,6$.
+
+n = 6
+xrange = -2.:1/256:0.25
+y = Array{Array{Float64,1}}(undef, n+1)
+labels = Array{String, 1}(undef, n+1)
+
+for i ∈ 0:n
+    q = c->fⁿ(Qc, 0.0, (c, 2^i))
```

Comment:
```suggestion
    q = c->fⁿ(Qc, 0.0, (c, 2*i))
```
para que sea la gráfica de q^{2n}